### PR TITLE
Update es.js

### PR DESCRIPTION
**Pull request by n-riesco**
* Added translation for `Click to enter radial axis title`
* Added translation for `new text`

### CODE REVIEW

File: lib/locales/es.js [hunk 1]
```diff
@@ -21,6 +21,7 @@ module.exports = {
         'Click to enter Plot title': 'Introducir el título de la Gráfica', // plot_api/plot_api.js:579
         'Click to enter X axis title': 'Introducir el título del eje X', // plots/plots.js:301
         'Click to enter Y axis title': 'Introducir el título del eje Y', // plots/plots.js:302
+        'Click to enter radial axis title': 'Introducir el título del eje radial',
```
Comment (n-riesco) [inline lib/locales/es.js]: Just nit-picking: here I've translated the original `radial axis`, but I think `polar axis` is more common.

File: lib/locales/es.js [hunk 2]
```diff
@@ -60,6 +61,7 @@ module.exports = {
         'mean:': 'media:',                                            // traces/box/calc.js:133
         'median:': 'mediana:',                                        // traces/box/calc.js:128
         'min:': 'mín:',                                               // traces/box/calc.js:129
+        'new text': 'nuevo texto',
```
Comment (n-riesco) [inline lib/locales/es.js]: Grammatically, the translation should be `texto nuevo` (as is `insertar un texto nuevo`), but in software `nuevo texto` is more common (I guess this is the result of the influence of the original English).

File: lib/locales/es.js (hunk 1 above)
Comment (alexcjohnson) [reply lib/locales/es.js]: Isn't `polar axis` ambiguous between angular and radial? Anyway that's a separate discussion, within the existing context this looks great, thanks!

File: lib/locales/es.js (hunk 2 above)
Comment (alexcjohnson) [reply lib/locales/es.js]: Insidious, isn't it 😈 

Thanks @n-riesco , I'll merge once I get tests to pass; the new `regl` code is awesome but it's wreaking havoc on CI right now...

File: lib/locales/es.js (hunk 1 above)
Comment (n-riesco) [reply lib/locales/es.js]: Oh, I've just realised that `radial axis` here is not the actual axis (i.e. the polar axis used as a reference to measure the angular coordinate).

**Comment by alexcjohnson** [comment]
cc @JonasGutermuth @jdugge - at some point we'll need a more formal process for communicating new translation keys, but for now since we only have a few languages this'll do. Care to translate "Click to enter radial axis title" (new with polar 2.0 #2200) and "new text" (for blank annotations, I missed this in my initial string wrapping push) to German?

**Pull request merged**